projects/sandbox/src/app/problem-viewer/problem-viewer.component.html

```diff
@@ -412,67 +412,51 @@ <h3 class="problem-viewer" *ngIf="ppd.problemPatternList.length === 0; else prob
     *ngIf="problemPatternsRecord && problemPatternsRecord.problemPatternList.length > 0"
   >
     <ng-container *ngIf="problemPatternsRecord.problemPatternList[0].recordAnalysisList.length > 0">
-      <div class="record-links-view" [ngClass]="{ closed: !recordLinksViewOpen && !isLoading }">
-        <span
-          class="record-links-view-opener"
-          libClickAware
-          (clickOutside)="recordLinksViewOpen = false"
-          [ignoreClasses]="ignoreClassesList"
-        >
-          <a
-            (click)="
-              openRecordLinksView(
-                problemPatternsRecord.problemPatternList[0].recordAnalysisList[0].recordId
-              )
-            "
-            class="nav-orb labelled element-orb allow-active-clicks"
-            [ngClass]="{
-              'is-active': recordLinksViewOpen,
-              'indicator-orb': isLoading,
-              spinner: isLoading
-            }"
-            title="record xml links (click to open)"
-          >
-          </a>
-        </span>
-        <div
-          class="record-links-view-content"
-          [ngClass]="{ disabled: !processedRecordData }"
-          libClickAware
-          (clickOutside)="recordLinksViewOpen = false"
-          [ignoreClasses]="ignoreClassesList"
-        >
-          <ul>
-            <li>
-              <ng-template #labelRefHarvest
-                ><span i18n="record link link@@recordLinkHarvest"
-                  >record before processing (as provided)</span
-                ></ng-template
-              >
-              <sb-copyable-link-item
-                [labelRef]="labelRefHarvest"
-                [href]="
-                  processedRecordData
-                    ? (problemPatternsRecord.datasetId
-                      | formatHarvestUrl: processedRecordData.europeanaRecordId)
-                    : ''
-                "
-              ></sb-copyable-link-item>
-            </li>
-            <li>
-              <ng-template #labelRefPortal
-                ><span i18n="record link link@@recordLinkPortal"
-                  >record after processing (as published)</span
-                ></ng-template
-              >
-              <sb-copyable-link-item
-                [labelRef]="labelRefPortal"
-                [href]="processedRecordData ? processedRecordData.portalRecordLink : ''"
-              ></sb-copyable-link-item>
-            </li>
-          </ul>
-        </div>
-      </div>
+      <sb-pop-out
```

Comment:
does this require a name or id?

Reply:
No.  The pop-out for the record-links is simpler and does not contain the title (close) link